src/opensynth/models/faraday/gaussian_mixture/gmm_lightning.py

```diff
@@ -36,6 +38,7 @@ def __init__(
         vae_module: pl.LightningModule,
```

Comment:
♻️

Reply:
As @charlotte-avery mentioned above, this is handled in https://github.com/OpenSynth-energy/OpenSynth/pull/47